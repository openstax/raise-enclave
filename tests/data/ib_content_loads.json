[{  
    "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
    "course_id": 0,
    "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
    "source_uri": "https://localhost:8000",
    "timestamp": 0,
    "eventname": "content_loaded_v1",
    "content_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
    "variant": "string"
}, {
    "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
    "course_id": 0,
    "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
    "source_uri": "https://localhost:8000",
    "timestamp": 0,
    "eventname": "content_loaded_v1",
    "content_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
    "variant": "string"
}]