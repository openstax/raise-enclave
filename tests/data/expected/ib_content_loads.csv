user_uuid,course_id,impression_id,timestamp,content_id,variant
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,3fa85f64-5717-4562-b3fc-2c963f66afa7,string
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,3fa85f64-5717-4562-b3fc-2c963f66afa7,string